### PR TITLE
Merge develop 8.2 17 10 2

### PR DESCRIPTION
**Pull request by montegoulding**
[resolution.txt](https://github.com/livecode/livecode/files/1347680/resolution.txt)

### CODE REVIEW

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 9e51efe

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 9e51efebf7123b16c78677b9a224c038ae2c614b

**Comment by livecode-vulcan** [comment]
😞 test failure 9e51efebf7123b16c78677b9a224c038ae2c614b

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/856)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/872)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/846)
* try-community-universal-ios-iphoneos11.0: [failure](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.0/builds/21)
* try-community-universal-ios-iphonesimulator11.0: [failure](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.0/builds/22)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/850)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/872)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/856)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/872)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/855)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/865)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/887)
* try-community-x86_64-win32: [pending](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/851)

File: ide-support/revdeploylibraryios.livecodescript [hunk 1]
```diff
@@ -39,16 +39,19 @@ function deployGetIphoneOSes
...
+      // Xcode 9.0 requires a Mac running Mac OS 10.12.6 or later
+      put "11.0,9.0" into tList[1]
    end if
    
    return tList
 end deployGetIphoneOSes
 
 function deployGetIosMinimumVersions
-   return "8.0,8.1,8.2,8.3,8.4,9.0,9.1,9.2,9.3,10.0,10.1,10.2,10.3"
+   return "8.0,8.1,8.2,8.3,8.4,9.0,9.1,9.2,9.3,10.0,10.1,10.2,11.0"
```
Comment (livecodepanos) [inline ide-support/revdeploylibraryios.livecodescript:54]: We need to keep both `10.3` and `11.0` here (it shows the deployment target in the iOS S/B settings, e.g `10.3 or later`, `11.0 or later` etc)

**Comment by livecodepanos** [comment]
We also need to build OpenSSL 1.1.0e prebuilts for iOS 11 (device and sim), as the `develop-8.x` branches have a previous OpenSSL version

File: prebuilt/fetch-libraries.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@ LIBS_win32=( OpenSSL Curl ICU CEF )
 LIBS_linux=( OpenSSL Curl ICU CEF )
 LIBS_emscripten=( ICU )
 
-SUBPLATFORMS_ios=(iPhoneSimulator8.2 iPhoneSimulator9.2 iPhoneSimulator10.2 iPhoneSimulator10.3 iPhoneOS9.2 iPhoneOS10.2 iPhoneOS10.3)
+SUBPLATFORMS_ios=(iPhoneSimulator8.2 iPhoneSimulator9.2 iPhoneSimulator10.2 iPhoneSimulator111.0 iPhoneOS9.2 iPhoneOS10.2 iPhoneOS11.0)
 SUBPLATFORMS_win32=(v140_static_debug v140_static_release)
```
Comment (livecodepanos) [inline prebuilt/fetch-libraries.sh:19]: There is a typo here iPhoneSimulator111.0 :)

**Comment by livecodepanos** [comment]
Replaced by https://github.com/livecode/livecode/pull/6019

**Pull request closed without merging**